internal/backends/winit/winitwindowadapter.rs

```diff
@@ -639,6 +653,25 @@ impl WindowAdapterSealed for WinitWindowAdapter {
     fn is_visible(&self) -> bool {
         self.winit_window().is_visible().unwrap_or(true)
     }
+
+    fn handle_focus_change(&self, _old: Option<ItemRc>, _new: Option<ItemRc>) {
+        #[cfg(enable_accesskit)]
```

Comment:
```suggestion
    #[cfg(enable_accesskit)]
    fn handle_focus_change(&self, _old: Option<ItemRc>, _new: Option<ItemRc>) {
```

(consistence with other functions)

Reply:
Thanks :)